app/frontend/package.json

```diff
@@ -18,19 +18,31 @@
   },
   "dependencies": {
     "@bcgov/bc-sans": "^1.0.1",
-    "@ckeditor/ckeditor5-build-classic": "^27.1.0",
+    "@ckeditor/ckeditor5-basic-styles": "^29.0.0",
+    "@ckeditor/ckeditor5-dev-utils": "^25.3.0",
+    "@ckeditor/ckeditor5-dev-webpack-plugin": "^25.3.0",
+    "@ckeditor/ckeditor5-editor-classic": "^29.0.0",
+    "@ckeditor/ckeditor5-essentials": "^29.0.0",
+    "@ckeditor/ckeditor5-link": "^29.0.0",
+    "@ckeditor/ckeditor5-paragraph": "^29.0.0",
+    "@ckeditor/ckeditor5-source-editing": "^29.0.0",
+    "@ckeditor/ckeditor5-theme-lark": "^29.0.0",
     "@ckeditor/ckeditor5-vue2": "^1.0.5",
     "axios": "^0.21.1",
     "core-js": "^3.10.1",
     "date-fns": "^2.21.1",
     "keycloak-js": "^12.0.4",
+    "moment": "^2.29.1",
```

Comment:
Can we avoid moment? It's legacy https://momentjs.com/docs/#/-project-status/
unless it's needed for some other plugin?

Reply:
Agreed on this. We should NOT be adding a direct package dependency on moment; if dependent packages need moment, they'll be importing it anyways (and we can't really control that). There are various alternatives for moment.js out there, depending on what aspects of date formatting/management you're using.
Ref: https://www.vhudyma-blog.eu/the-best-momentjs-alternatives/

Reply:
yeah i know. but a lot of the code from mssc used moment. 
i tried using the date-fns library (which i think is a good alternative) but its quite a lot of work to switch to that.

Reply:
replaced mostly with date-fns library.